### PR TITLE
Plugins: Add plugin settings DTO

### PR DESCRIPTION
**Pull request by wbrowne**
**What this PR does / why we need it**:

Migrating from `models` package to dedicated `pluginsettings` package and simplifying service usage

`models.PluginSettingInfoDTO` ➡️  `pluginsettings.DTO`
`models.PluginSetting` ➡️   `pluginsettings.DTO`
`models.UpdatePluginSettingVersionCmd` ➡️   `pluginsettings.UpdatePluginVersionArgs`
`models.UpdatePluginSettingCmd` ➡️   `pluginsettings.UpdateArgs`
`models.GetPluginSettingByIdQuery` ➡️  `pluginsettings.GetByPluginIDArgs`

### CODE REVIEW

File: pkg/services/plugindashboards/service/dashboard_updater.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ type DashboardUpdater struct {
 func (du *DashboardUpdater) updateAppDashboards() {
 	du.logger.Debug("Looking for app dashboard updates")
 
-	pluginSettings, err := du.pluginSettingsService.GetPluginSettings(context.Background(), 0)
+	pluginSettings, err := du.pluginSettingsService.GetPluginSettings(context.Background(), &pluginsettings.GetArgs{OrgID: 0})
```
Comment (marefr) [inline pkg/services/plugindashboards/service/dashboard_updater.go:57]: Nit. Maybe consider `GetAllPluginSettings` or similar without args? Then the consumer doesn't have to provider orgId=0 and logic for "all" can be isolated in plugin settings service.

**Review by marefr — APPROVED** [review]
LGTM I think, great :+1: Added a couple comments

File: pkg/services/plugindashboards/service/dashboard_updater.go (hunk 1 above)
Comment (wbrowne) [reply pkg/services/plugindashboards/service/dashboard_updater.go:57]: (Discussed offline) Will follow up with this if necessary

File: pkg/services/sqlstore/plugin_setting.go [hunk 2]
```diff
@@ -7,25 +7,22 @@ import (
...
+	err := ss.WithDbSession(ctx, func(sess *DBSession) error {
+		if orgID != 0 {
+			sess.In("org_id", orgID)
+		}
 
-	if orgID != 0 {
-		sql += "WHERE org_id=?"
-		params = append(params, orgID)
-	}
+		err := sess.Find(&rslt)
```
[the remark this answers is not in the document]
Comment (wbrowne) [reply pkg/services/sqlstore/plugin_setting.go:17]: (Discussed offline) Will follow up with this if necessary

**Pull request merged**